example/ipsec_crypto/odp_ipsec_stream.c

```diff
@@ -65,6 +65,14 @@ void init_stream_db(void)
 	memset(stream_db, 0, sizeof(*stream_db));
 }
 
+void free_stream_db_intf(void)
```

Comment:
The code would be cleaner if this was encapsulated in something like deinit_stream_db(), which would be the opposite of init_stream_db(). That function should then take care of freeing the SHM too.